ui/dashboards/src/components/AddPanelButton/AddPanelButton.tsx

```diff
@@ -11,19 +11,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { Button } from '@mui/material';
+import { Button, ButtonProps } from '@mui/material';
 import AddPanelIcon from 'mdi-material-ui/ChartBoxPlusOutline';
 import { InfoTooltip } from '@perses-dev/components';
 import { TOOLTIP_TEXT } from '../../constants';
 import { useDashboardActions } from '../../context';
 
-export const AddPanelButton = () => {
+export interface AddPanelButtonProps extends Pick<ButtonProps, 'fullWidth'> {
+  /**
```

Comment:
Adding some variation props to support the empty state use case for these buttons.